apps/token/contracts/Token.sol

```diff
@@ -21,6 +21,8 @@ contract Token is ERC20Token, AragonApp {
   }
 
   function mintFor(address contributorAccount, uint256 amount, uint32 contributionId) public isInitialized auth(MINT_TOKEN_ROLE) {
+    require(amount > 0, "amount should be greater than zero");
```

Comment:
I think so far I always used keys like `INVALID_AMOUNT` instead of full messages. 
This should make the code smaller and is enough to identify the error and potentially also map it to a full message